packs/vsphere/actions/workflows/vm_hw_basic_build.yaml

```diff
@@ -38,12 +39,13 @@ vsphere.vm_hw_basic_build:
         description: <% $.vm_description %>
         vsphere: <% $.vsphere %>
       publish:
-        vm_id: <% $.barebones_build.result.vm_id %>
+        vm_id: <% task(barebones_build).result.result.vm_id %>
       on-success:
         - add_controller
     add_controller:
       # [186, 198]
       action: vsphere.vm_hw_scsi_controller_add
+      wait-after: 12
```

Comment:
they were added to spread the load when the workflow is called concurrently.
In my setup i can be calling it concurrently 10 times which on a single instance taxes mistral a lot. the waits allow things to run a bit more smoothly by giving actions time to complete in parrallel before the next step of the concurrent flows start.